pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -126,7 +142,7 @@ stdenv.mkDerivation ({
   # derivation. Next, patch the first line to use the escript that will be
   # built during the build phase of this derivation. Finally, building the
   # documentation requires the erlang-logo.png asset.
-  preConfigure = ''
+  preConfigure = optionals (lib.strings.versionOlder version "22.0") ''
```

Comment:
```suggestion
  preConfigure = optionals (lib.strings.versionOlder version "22.0") ''
```

There is a problem here:

![image](https://github.com/user-attachments/assets/35fadca3-6336-4d72-a481-92c328b34a5e)

Reply:
Please squash my commit. I've only commited to re-start eval. Such as we know what else is wrong here.

Reply:
nixpkgs doesn't support anything older than 24, so this section is removed.